npm/pod.go

```diff
@@ -27,7 +27,8 @@ func isInvalidPodUpdate(oldPodObj, newPodObj *corev1.Pod) (isInvalidUpdate bool)
 		oldPodObj.Status.Phase == newPodObj.Status.Phase &&
 		oldPodObj.Status.PodIP == newPodObj.Status.PodIP &&
 		newPodObj.ObjectMeta.DeletionTimestamp == nil &&
-		newPodObj.ObjectMeta.DeletionGracePeriodSeconds == nil
+		newPodObj.ObjectMeta.DeletionGracePeriodSeconds == nil &&
+		newPodObj.Spec.HostNetwork //Ignore if HostNetwork pod
```

Comment:
Should we use && here? In my understanding, it's an OR relationship since if it use a host network, then we don't need to check other condition, we can just ignore it and let it be an invalid update.

Reply:
Thank you.